doc-content/enterprise-only/installation/clustering-bc-configure-domain-proc.adoc

```diff
@@ -40,13 +40,17 @@ These steps describe a basic cluster configuration. For more complex configurati
   <property name="appformer-jms-connection-mode" value="JNDI"/>
   <property name="appformer-jms-username" value="<JMS_USER>  "/>
   <property name="appformer-jms-password" value="<JMS_USER_PASSWORD>"/>
-  <property name="org.appformer.ext.metadata.elastic.port" value="9300"/>
-  <property name="org.appformer.ext.metadata.elastic.host" 
-      value="<ELASTICSEARCH_NODE_IP>"/>
-  <property name="org.appformer.ext.metadata.elastic.cluster" 
+  <property name="org.appformer.ext.metadata.infinispan.port" value="11222"/>
+  <property name="org.appformer.ext.metadata.infinispan.host" 
+      value="<INFINISPAN_NODE_IP>"/>
+  <property name="org.appformer.ext.metadata.infinispan.username"
+      value="<USERNAME>"/>
+  <property name="org.appformer.ext.metadata.infinispan.password"
+      value="<PASSWORD>"/>
+  <property name="org.appformer.ext.metadata.infinispan.cluster" 
       value="kie-cluster"/>
-  <property name="org.appformer.ext.metadata.index" value="elastic"/> 
-  <property name="org.appformer.ext.metadata.elastic.retries" value="10"/>
+  <property name="org.appformer.ext.metadata.index" value="infinispan"/> 
+  <property name="org.appformer.ext.metadata.infinispan.retries" value="10"/>
```

Comment:
Here I forgot to tell you that Infinispan connection doesn't have retries, so we can delete this line

Reply:
Retry line removed.